### ISSUE TITLE
SeatAccess.polkit failed to start hyprland

### ISSUE THREAD

**Issue by alpenstorm** [archlinux/archinstall#2914]
### Which ISO version are you using?

2024-11-01

### The installation log

```json
Hardware model detected: Micro-Star International Co., Ltd. Katana GF66 12UE; UEFI mode: True
Processor model detected: 12th Gen Intel(R) Core(TM) i7-12700H
Memory statistics: 38897008 available out of 40771128 total installed
Could not detect virtual system: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [1]: none

System is not running in a VM: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [1]: none

Virtualization detected: None; is VM: False
Graphics devices detected: dict_keys(['Intel Corporation Alder Lake-P GT2 [Iris Xe Graphics] (rev 0c)', 'NVIDIA Corporation GA106M [GeForce RTX 3060 Mobile / Max-Q] (rev a1)', 'Sandisk Corp SanDisk Ultra 3D / WD Blue SN550 NVMe SSD (rev 01)'])
Disk states before installing: [
    {
        "children": [],
        "fsavail": 0,
        "fsroots": [
            "."
        ],
        "fstype": "squashfs",
        "fsuse_percentage": "100%",
        "fsver": "4.0",
        "log_sec": 512,
        "mountpoint": "/run/archiso/airootfs",
        "mountpoints": [
            "/run/archiso/airootfs"
        ],
        "name": "loop0",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/loop0",
        "pkname": null,
        "pttype": null,
        "ptuuid": null,
        "rota": false,
        "size": "810 MiB",
        "tran": null,
        "type": "loop",
        "uuid": null
    },
    {
        "children": [
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "iso9660",
                "fsuse_percentage": null,
                "fsver": "Joliet Extension",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "sda1",
                "partn": 1,
                "parttype": "0x0",
                "partuuid": "211019da-01",
                "path": "/dev/sda1",
                "pkname": "sda",
                "pttype": "dos",
                "ptuuid": "211019da",
                "rota": true,
                "size": "978 MiB",
                "tran": null,
                "type": "part",
                "uuid": "2024-11-01-10-09-22-00"
            },
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "vfat",
                "fsuse_percentage": null,
                "fsver": "FAT32",
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "sda2",
                "partn": 2,
                "parttype": "0xef",
                "partuuid": "211019da-02",
                "path": "/dev/sda2",
                "pkname": "sda",
                "pttype": "dos",
                "ptuuid": "211019da",
                "rota": true,
                "size": "175 MiB",
                "tran": null,
                "type": "part",
                "uuid": "6724-A8D2"
            }
        ],
        "fsavail": null,
        "fsroots": [],
        "fstype": "iso9660",
        "fsuse_percentage": null,
        "fsver": "Joliet Extension",
        "log_sec": 512,
        "mountpoint": null,
        "mountpoints": [],
        "name": "sda",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/sda",
        "pkname": null,
        "pttype": "dos",
        "ptuuid": "211019da",
        "rota": true,
        "size": "15000 MiB",
        "tran": "usb",
        "type": "disk",
        "uuid": "2024-11-01-10-09-22-00"
    },
    {
        "children": [
            {
                "children": [],
                "fsavail": null,
                "fsroots": [],
                "fstype": "ntfs",
                "fsuse_percentage": null,
                "fsver": null,
                "log_sec": 512,
                "mountpoint": null,
                "mountpoints": [],
                "name": "nvme1n1p1",
                "partn": 1,
                "parttype": "ebd0a0a2-b9e5-4433-87c0-68b6b72699c7",
                "partuuid": "a7338747-990a-46a8-98aa-da98e4343043",
                "path": "/dev/nvme1n1p1",
                "pkname": "nvme1n1",
                "pttype": "gpt",
                "ptuuid": "00cfc434-c9c9-41aa-9b3b-ecc99ab45209",
                "rota": false,
                "size": "953866 MiB",
                "tran": "nvme",
                "type": "part",
                "uuid": "605A669A5A666D2A"
            }
        ],
        "fsavail": null,
        "fsroots": [],
        "fstype": null,
        "fsuse_percentage": null,
        "fsver": null,
        "log_sec": 512,
        "mountpoint": null,
        "mountpoints": [],
        "name": "nvme1n1",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/nvme1n1",
        "pkname": null,
        "pttype": "gpt",
        "ptuuid": "00cfc434-c9c9-41aa-9b3b-ecc99ab45209",
        "rota": false,
        "size": "953869 MiB",
        "tran": "nvme",
        "type": "disk",
        "uuid": null
    },
    {
        "children": [],
        "fsavail": null,
        "fsroots": [],
        "fstype": null,
        "fsuse_percentage": null,
        "fsver": null,
        "log_sec": 512,
        "mountpoint": null,
        "mountpoints": [],
        "name": "nvme0n1",
        "partn": null,
        "parttype": null,
        "partuuid": null,
        "path": "/dev/nvme0n1",
        "pkname": null,
        "pttype": null,
        "ptuuid": null,
        "rota": false,
        "size": "488386 MiB",
        "tran": "nvme",
        "type": "disk",
        "uuid": null
    }
]
Checking version...
Failed determine pacman version: ['/usr/bin/pacman', '-Qu', 'archinstall'] exited with abnormal exit code [1]: [?25l[?25h
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/__init__.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/custom.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktop.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/minimal.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/server.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/tailored.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/xorg.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/applications/__init__.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/applications/pipewire.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/__init__.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/awesome.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/bspwm.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/budgie.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/cinnamon.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/cosmic.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/cutefish.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/deepin.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/enlightenment.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/gnome.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/hyprland.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/i3.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/lxqt.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/mate.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/plasma.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/qtile.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/sway.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/desktops/xfce4.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/servers/__init__.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/servers/cockpit.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/servers/docker.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/servers/httpd.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/servers/lighttpd.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/servers/mariadb.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/servers/nginx.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/servers/postgresql.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/servers/sshd.py
Importing profile: /usr/lib/python3.12/site-packages/archinstall/default_profiles/servers/tomcat.py
 -- Chosen configuration --
{
    "additional-repositories": [
        "multilib"
    ],
    "archinstall-language": "English",
    "audio_config": {
        "audio": "pipewire"
    },
    "bootloader": "Systemd-boot",
    "config_version": "3.0.0",
    "debug": false,
    "disk_config": {
        "config_type": "default_layout",
        "device_modifications": [
            {
                "device": "/dev/nvme0n1",
                "partitions": [
                    {
                        "btrfs": [],
                        "dev_path": null,
                        "flags": [
                            "Boot",
                            "ESP"
                        ],
                        "fs_type": "fat32",
                        "mount_options": [],
                        "mountpoint": "/boot",
                        "obj_id": "e6b0d61b-f9f9-4fe5-81be-52dabc7895f9",
                        "size": {
                            "sector_size": {
                                "unit": "B",
                                "value": 512
                            },
                            "unit": "GiB",
                            "value": 1
                        },
                        "start": {
                            "sector_size": {
                                "unit": "B",
                                "value": 512
                            },
                            "unit": "MiB",
                            "value": 1
                        },
                        "status": "create",
                        "type": "primary"
                    },
                    {
                        "btrfs": [],
                        "dev_path": null,
                        "flags": [],
                        "fs_type": "btrfs",
                        "mount_options": [
                            "compress=zstd"
                        ],
                        "mountpoint": "/",
                        "obj_id": "21615d0a-44e7-46b7-9770-d5137c6298e3",
                        "size": {
                            "sector_size": {
                                "unit": "B",
                                "value": 512
                            },
                            "unit": "B",
                            "value": 511034351616
                        },
                        "start": {
                            "sector_size": {
                                "unit": "B",
                                "value": 512
                            },
                            "unit": "B",
                            "value": 1074790400
                        },
                        "status": "create",
                        "type": "primary"
                    }
                ],
                "wipe": true
            }
        ]
    },
    "disk_encryption": null,
    "hostname": "archlinux",
    "kernels": [
        "linux"
    ],
    "locale_config": {
        "kb_layout": "us",
        "sys_enc": "UTF-8",
        "sys_lang": "en_US"
    },
    "mirror_config": null,
    "network_config": {
        "type": "nm"
    },
    "no_pkg_lookups": false,
    "ntp": true,
    "offline": false,
    "packages": [
        "nano",
        "vim",
        "firefox",
        "git"
    ],
    "parallel downloads": 0,
    "profile_config": {
        "gfx_driver": "Nvidia (proprietary)",
        "greeter": "lightdm-slick-greeter",
        "profile": {
            "custom_settings": {
                "Hyprland": {
                    "seat_access": "polkit"
                }
            },
            "details": [
                "Hyprland"
            ],
            "main": "Desktop"
        }
    },
    "save_config": null,
    "script": "guided",
    "silent": false,
    "skip_ntp": false,
    "skip_version_check": false,
    "swap": true,
    "timezone": "America/New_York",
    "uki": false,
    "version": "3.0.0"
}
Unmounting all existing partitions: /dev/nvme0n1
Wiping partitions and metadata: /dev/nvme0n1
Creating partitions: /dev/nvme0n1
	Type: primary
	Filesystem: fat32
	Geometry: 2048 start sector, 2097152 length
	Type: primary
	Filesystem: btrfs
	Geometry: 2099200 start sector, 998113968 length
Formatting filesystem: mkfs.fat -F 32 /dev/nvme0n1p1
partition information found: {'name': 'nvme0n1p1', 'path': '/dev/nvme0n1p1', 'pkname': 'nvme0n1', 'size': '1024 MiB', 'log_sec': 512, 'pttype': 'gpt', 'ptuuid': '2123fe3b-5c9a-429c-8792-ccda9cabeb6f', 'rota': False, 'tran': 'nvme', 'partn': 1, 'partuuid': '21314434-f196-4608-b621-dbcad36c608f', 'parttype': 'c12a7328-f81f-11d2-ba4b-00a0c93ec93b', 'uuid': '33B1-687B', 'fstype': 'vfat', 'fsver': 'FAT32', 'fsavail': None, 'fsuse_percentage': None, 'type': 'part', 'mountpoint': None, 'mountpoints': [], 'fsroots': [], 'children': []}
Formatting filesystem: mkfs.btrfs -f /dev/nvme0n1p2
partition information found: {'name': 'nvme0n1p2', 'path': '/dev/nvme0n1p2', 'pkname': 'nvme0n1', 'size': '487360 MiB', 'log_sec': 512, 'pttype': 'gpt', 'ptuuid': '2123fe3b-5c9a-429c-8792-ccda9cabeb6f', 'rota': False, 'tran': 'nvme', 'partn': 2, 'partuuid': '6b257e62-2218-4a9b-a254-8167afd1fae0', 'parttype': '4f68bce3-e8cd-4db1-96e7-fbcaf984b709', 'uuid': 'c747a64b-d9d6-4724-a6b8-12ec1a574f8a', 'fstype': 'btrfs', 'fsver': None, 'fsavail': None, 'fsuse_percentage': None, 'type': 'part', 'mountpoint': None, 'mountpoints': [], 'fsroots': [], 'children': []}
Creating subvolumes: /dev/nvme0n1p2
Mounting /dev/nvme0n1p2: mount -o compress=zstd /dev/nvme0n1p2 /mnt/arch_btrfs
Partition /dev/nvme0n1p2 is currently mounted at: ['/mnt/arch_btrfs']
Unmounting mountpoint: /mnt/arch_btrfs
Starting installation...
Mounting ordered layout
No lvm config defined to be mounted
Mounting partition layout
Mounting /dev/nvme0n1p2: mount -o compress=zstd /dev/nvme0n1p2 /mnt/archinstall
Mounting /dev/nvme0n1p1: mount /dev/nvme0n1p1 /mnt/archinstall/boot
Waiting for time sync (timedatectl show) to complete.
Waiting for automatic mirror selection (reflector) to complete.
Waiting for Arch Linux keyring sync (archlinux-keyring-wkd-sync) to complete.
System is not running in a VM: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [1]: none

The multilib flag is set. This system will be installed with the multilib repository enabled.
The testing flag is not set. This system will be installed without testing repositories enabled.
Installing packages: ['base', 'base-devel', 'linux-firmware', 'linux', 'btrfs-progs', 'intel-ucode']
Setting keyboard language to us
Keyboard language for this installation is now set to: us
Setting up swap on zram
Installing packages: ['zram-generator']
Enabling service systemd-zram-setup@zram0.service
Adding bootloader Systemd-boot to /dev/nvme0n1p1
Installing systemd bootloader
Installing packages: ['efibootmgr']
Identifying root partition by PARTUUID: 6b257e62-2218-4a9b-a254-8167afd1fae0
kernel parameters: root=PARTUUID=6b257e62-2218-4a9b-a254-8167afd1fae0 zswap.enabled=0 rw rootfstype=btrfs
Installing packages: ['networkmanager']
Installing packages: ['network-manager-applet']
Enabling service NetworkManager.service
Creating user alpen
Setting password for alpen
Enabling sudo permissions for alpen
Installing audio server: Pipewire
Installing packages: ['pipewire', 'pipewire-alsa', 'pipewire-jack', 'pipewire-pulse', 'gst-plugin-pipewire', 'libpulse', 'wireplumber']
Installing packages: ['nano', 'vim', 'firefox', 'git']
Installing packages: ['nano', 'vim', 'openssh', 'htop', 'wget', 'iwd', 'wireless_tools', 'wpa_supplicant', 'smartmontools', 'xdg-utils']
Installing profile Hyprland...
Installing packages: ['hyprland', 'dunst', 'kitty', 'dolphin', 'wofi', 'xdg-desktop-portal-hyprland', 'qt5-wayland', 'qt6-wayland', 'polkit-kde-agent', 'grim', 'slurp']
Enabling service SeatAccess.polkit
Unable to start service SeatAccess.polkit: ['/usr/bin/arch-chroot', '/mnt/archinstall', 'systemctl', 'enable', 'SeatAccess.polkit'] exited with abnormal exit code [1]: 
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/archinstall/lib/installer.py", line 621, in enable_service
    self.arch_chroot(f'systemctl enable {service}')
  File "/usr/lib/python3.12/site-packages/archinstall/lib/installer.py", line 636, in arch_chroot
    return self.run_command(cmd)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/archinstall/lib/installer.py", line 630, in run_command
    return SysCommand(f'/usr/bin/arch-chroot {self.target} {cmd}')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/archinstall/lib/general.py", line 369, in __init__
    self.create_session()
  File "/usr/lib/python3.12/site-packages/archinstall/lib/general.py", line 418, in create_session
    with SysCommandWorker(
         ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/archinstall/lib/general.py", line 199, in __exit__
    raise SysCallError(
archinstall.lib.exceptions.SysCallError: ['/usr/bin/arch-chroot', '/mnt/archinstall', 'systemctl', 'enable', 'SeatAccess.polkit'] exited with abnormal exit code [1]: 

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/archinstall/__init__.py", line 337, in run_as_a_module
    main()
  File "/usr/lib/python3.12/site-packages/archinstall/__init__.py", line 330, in main
    importlib.import_module(mod_name)
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/usr/lib/python3.12/site-packages/archinstall/scripts/guided.py", line 193, in <module>
    guided()
  File "/usr/lib/python3.12/site-packages/archinstall/scripts/guided.py", line 190, in guided
    perform_installation(archinstall.storage.get('MOUNT_POINT', Path('/mnt')))
  File "/usr/lib/python3.12/site-packages/archinstall/scripts/guided.py", line 59, in perform_installation
    with Installer(
         ^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/archinstall/lib/installer.py", line 114, in __exit__
    raise exc_val
  File "/usr/lib/python3.12/site-packages/archinstall/scripts/guided.py", line 124, in perform_installation
    profile_handler.install_profile_config(installation, profile_config)
  File "/usr/lib/python3.12/site-packages/archinstall/lib/profile/profiles_handler.py", line 241, in install_profile_config
    profile.install(install_session)
  File "/usr/lib/python3.12/site-packages/archinstall/default_profiles/desktop.py", line 103, in install
    install_session.enable_service(profile.services)
  File "/usr/lib/python3.12/site-packages/archinstall/lib/installer.py", line 623, in enable_service
    raise ServiceException(f"Unable to start service {service}: {err}")
archinstall.lib.exceptions.ServiceException: Unable to start service SeatAccess.polkit: ['/usr/bin/arch-chroot', '/mnt/archinstall', 'systemctl', 'enable', 'SeatAccess.polkit'] exited with abnormal exit code [1]: 

Archinstall experienced the above error. If you think this is a bug, please report it to
https://github.com/archlinux/archinstall and include the log file "/var/log/archinstall/install.log".

Hint: To extract the log from a live ISO 
curl -F'file=@/var/log/archinstall/install.log' https://0x0.st
```

### describe the problem
using nvidia proprietary drivers and polkit installed before running the script

**Comment by svartkanin** [comment]
This has been fixed already

**Comment by svartkanin** [comment]
Because it's not released yet, you can use the latest master

**Comment by alpenstorm** [comment]
how?

**Comment by Epirius** [comment]
you can run
```
sudo pacman -S git
git clone https://github.com/archlinux/archinstall.git
cd archinstall
python -m archinstall
```

**Comment by svartkanin** [comment]
New version also just got released 3.0.1
